setup/unattended-install/unattended-install.php

```diff
@@ -217,8 +219,16 @@ function PrintUsageAndExit()
 			die("Cleanup not implemented for a partial database (prefix= '$sDBPrefix')\nExiting.");
 		}
 
-		$oMysqli = new mysqli($sDBServer, $sDBUser, $sDBPwd);
-		if ($oMysqli->connect_errno)
+		$oMysqli = new mysqli();
+		$iMySqlFlag = 0;
+		if ($bDBTlsEnabled)
+		{
+			$iMySqlFlag = (empty($sDBTlsCa))
+				? MYSQLI_CLIENT_SSL_DONT_VERIFY_SERVER_CERT
+				: MYSQLI_CLIENT_SSL;
+			$oMysqli->ssl_set($bDBTlsEnabled, null, $sDBTlsCa, null, null);
+		}
+		if ($oMysqli->real_connect($sDBServer, $sDBUser, $sDBPwd, null, null, null, $iMySqlFlag))
```

Comment:
Now you made a mistake :smile:
```suggestion
		if (!$oMysqli->real_connect($sDBServer, $sDBUser, $sDBPwd, null, null, null, $iMySqlFlag))
```

Reply:
😅